atc/metric/emitter/newrelic.go

```diff
@@ -173,16 +144,117 @@ func (emitter *NewRelicEmitter) Emit(logger lager.Logger, event metric.Event) {
 	// otherwise recording it. (This won't be easily graphable, that's okay,
 	// this is more for monitoring synthetics)
 	if event.State != metric.EventStateOK {
-		singlePayload := emitter.simplePayload(logger, event, "alert")
+		singlePayload := emitter.transformToNewRelicEvent(logger, event, "alert")
 		// We don't have friendly names for all the metrics, and part of the
 		// point of this alert is to catch events we should be logging but
 		// didn't; therefore, be consistently inconsistent and use the
 		// concourse metric names, not our translation layer.
 		singlePayload["metric"] = event.Name
-		payload = append(payload, singlePayload)
+		emitter.NewRelicBatch = append(emitter.NewRelicBatch, singlePayload)
+	}
+
+	duration := time.Since(emitter.LastEmitTime)
+	if len(emitter.NewRelicBatch) >= emitter.BatchSize || duration >= emitter.BatchDuration {
+		logger.Debug("pre-emit-batch", lager.Data{
```

Comment:
oh this can be sooo chatty - do you think this is necessary? or was this something used during dev that was just left?

Reply:
This was copied from the influxdb emitter 😬